core/amber/requirements.txt

```diff
@@ -10,7 +10,7 @@ pytest==7.4.0
 python-dateutil==2.8.2
 pytest-timeout==2.2.0
 protobuf==3.20.3
-betterproto==2.0.0b3
+betterproto==2.0.0b6
```

Comment:
I had an impression the latest is b7. do we want to change it to b7?

Reply:
updated to b7.